universal-login-web3/test/ULWeb3Provider.test.ts

```diff
@@ -76,7 +76,7 @@ describe('ULWeb3Provider', () => {
   describe('get accounts', () => {
     it('initialize onboarding when wallet does not exist', async () => {
       const initOnboardingSpy = sinon.spy(() => null);
-      sinon.replace(ulProvider, 'initOnboarding', initOnboardingSpy);
+      sinon.replace(ulProvider, 'initOnboarding', initOnboardingSpy as any);
```

Comment:
Why is this needed?

Reply:
I resolved it. Initialization should return a Promise, not null